src/utils/isBase64UrlImage.ts

```diff
@@ -0,0 +1,12 @@
+export const isBase64UrlImage = async (base64String: string) => {
```

Comment:
I'm missing a return type here. The await on the returned Promise makes it a bit of a head scratcher I think.